frontend/src/core/hooks/useStream.js

```diff
@@ -132,7 +147,9 @@ const useStream = (q) => {
         retry: 2,
         onSuccess: (newEvents) => {
           if (newEvents && newEvents.stream_boundary && newEvents.events) {
-            setEvents([...newEvents.events, ...events]);
+            queryClient
```

Comment:
Use setQueryData to manipulate on query cache data objects. 
Good reference: 
https://github.com/bugout-dev/search/blob/master/frontend/src/core/hooks/useDeleteJournal.js